CHANGELOG.md

```diff
@@ -2,14 +2,16 @@
 
 ## master / unreleased
 
+## 1.12.0 in pgoress
+
 * [CHANGE] Changed default for `-ingester.min-ready-duration` from 1 minute to 15 seconds. #4539
-* [CHANGE] query-frontend: Do not print anything in the logs of `query-frontend` if a in-progress query has been canceled (context canceled). #4562
+* [CHANGE] query-frontend: Do not print anything in the logs of `query-frontend` if a in-progress query has been canceled (context canceled) to avoid spam. #4562
```

Comment:
Maybe "noisy logs" or "spam in logs", since "spam" normally relates to email.

Reply:
Right. Thanks for the suggestion. I will adopt "noisy logs" because I was iffy about "spam "because: 

![image](https://user-images.githubusercontent.com/787449/154670004-5dded83f-753b-4a00-98b3-766ab768555a.png)